src/init/starship.zsh

```diff
@@ -64,4 +64,6 @@ zle -N zle-keymap-select
 export STARSHIP_SHELL="zsh"
 
 # Set up the session key that will be used to store logs
-export STARSHIP_SESSION_KEY=$(::STARSHIP:: session)
+STARSHIP_SESSION_KEY="$RANDOM$RANDOM$RANDOM$RANDOM$RANDOM"; # Random generates a number b/w 0 - 32767
+STARSHIP_SESSION_KEY="${STARSHIP_SESSION_KEY}0000000000000000" # Pad it to 16+ chars.
```

Comment:
I don't think the padding logic is really needed. It should be fine if the session key doesn't always have the same length.

Reply:
Maybe a personal peeve but I like uniformity and seeing 'N' files of same file-name style beats a bunch of random ones 😅 No strong opinion though. I can remove this if you prefer to simplify this.